lib/fathead/nodejs_ref/parse.js

```diff
@@ -28,7 +28,7 @@ function getAbstract(obj) {
         stability = '<p>Stability: ' + obj.stability + ' - ' + obj.stabilityText + '</p>';
     }
 
-    a = code + a + stability;
+    a = '<section class="prog__container">' a + code + stability + '</section>';
```

Comment:
looks like you have a typo here? Should it be `'<section class="prog__container">' + a + code` ?

After changing that I'm also getting another error:

```
/mnt/ebs/usr/local/ddh/zeroclickinfo-fathead/lib/fathead/nodejs_ref/parse.js:108
    json[sectionKey].forEach(function(node) {
                    ^

TypeError: Cannot read property 'forEach' of undefined
    at /mnt/ebs/usr/local/ddh/zeroclickinfo-fathead/lib/fathead/nodejs_ref/parse.js:108:21
    at Array.forEach (native)
    at Object.<anonymous> (/mnt/ebs/usr/local/ddh/zeroclickinfo-fathead/lib/fathead/nodejs_ref/parse.js:107:13)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Function.Module.runMain (module.js:441:10)
    at startup (node.js:139:18)
    at node.js:974:3
```